products/BellHybrid/apps/application-bell-settings/models/DateTimeUnitsModel.cpp

```diff
@@ -173,6 +173,15 @@ namespace app::bell_settings
         application->bus.sendUnicast(std::move(msg), service::name::service_time);
     }
 
+    auto DateTimeUnitsModel::isDateChanged() -> bool
+    {
+        const auto date = daySetListItem->dateSetSpinner->getDate();
+        if (dateLoaded != date) {
```

Comment:
You can just `return dateLoaded != date;` :wink:

Reply:
Done